src/core/reopt_inputs.jl

```diff
@@ -190,14 +190,15 @@ function REoptInputs(s::AbstractScenario)
     months = 1:12
 
     levelization_factor, pwf_e, pwf_om, pwf_fuel, pwf_emissions_cost, pwf_grid_emissions, third_party_factor, pwf_offtaker, pwf_owner = setup_present_worth_factors(s, techs)
+    print("\nlevelization_factor: ", levelization_factor)
```

Comment:
Remove if this was a debugging print. Also FYI "println()" will do what the "\n" does (I think).